README.md

```diff
@@ -73,6 +74,34 @@ The `pocketbase-async` package was inspired and guided in implementation by seve
 
 Furthermore, a lot of the API tests were adapted from Vaphes' work (licensed MIT).
 
+## Development
```

Comment:
We indeed probably need a section like this in the to be written documentation, maybe not in the readme.md